### PR TITLE
Nwb refactor metadata

### PR DESCRIPTION
**Pull request by luiztauffer**
Uses metadata dictionary info to build nwb file

### CODE REVIEW

**Comment by bendichter** [comment]
Looks good so far!

File: spikeextractors/extractors/nwbextractors/nwbextractors.py [hunk 1]
```diff
@@ -247,15 +279,15 @@ def get_channel_ids(self):
         return self.channel_ids.tolist()
 
     @staticmethod
-    def write_recording(recording, save_path, acquisition_name='ElectricalSeries', **nwbfile_kwargs):
+    def write_recording(recording, save_path, metadata={}):
```
Comment (bendichter) [inline spikeextractors/extractors/nwbextractors/nwbextractors.py]: I prefer `metadata=None` and then having 

```
if metadata is None:
    metadata = dict()
```

I know it's a little extra code, but having a mutable as an input arg can lead to some weird behavior I'd rather avoid

**Review by bendichter — COMMENTED** [review]
Looks good. Will this interact with the GUI as is?

**Comment by luiztauffer** [comment]
the metadata is in the same format as produced by the GUI.
However, the GUI only interfaces with a `conversion_function()` provided by the user, it doesn't call spikeextract directly.

File: spikeextractors/extractors/nwbextractors/nwbextractors.py (hunk 1 above)
Comment (luiztauffer) [reply spikeextractors/extractors/nwbextractors/nwbextractors.py]: done here:
135a003f710518f961fa4464b859bf340c33055c

**Pull request merged**